Docs/source/Overview/design-requirements.rst

```diff
@@ -0,0 +1,28 @@
+Design Requirements
+====================
+Read the regulations at `American Solar Challenge <https://www.americansolarchallenge.org/regulations/2020-american-solar-challenge-regulations/>`_ before creating a BPS. 
+
+Design requirements vary based on the type of batteries you use. For example, lead-acid packs require less protection than lithium-ion cells. BeVolt’s battery pack uses Samsung INR18650-30Q lithium-ion cells. The BPS is required to monitor the current, voltage, and temperature of the battery pack.
+
+.. 
+    Packs vs cells - do lead-acid batteries only come as single units or can you get lead-acid cells? Can a battery pack contain one of either li-ion cells or lead-acid cells? If so, 
```

Comment:
Lithium-ion and lead-acid batteries both have cells. Multiple cells are combined to create a pack. I see the issue here where we are comparing lithium-ion **cells** to lead-acid **packs.** The idea we are trying to get across is that a lithium-ion battery is less stable than a lead-acid battery, so you need more protection for lithium-ion than lead-acid.